Header header

EngineServoState basic_state

uint8 errors_cnt # of accidents
uint8 errors_flag1 # error code 
uint8 errors_flag2 # error code 
